src/test/java/weblogic/logging/exporter/config/MapUtilsTest.java

```diff
@@ -41,6 +44,60 @@ public void whenStringArrayValueIsList_returnAsArray() {
     assertThat(MapUtils.getStringArray(map, "values"), arrayContaining("7", "8", "true"));
   }
 
+  @DisplayName("Check getBooleanValue for true values")
+  @Test
+  public void checkGetBooleanForTrueValues() {
```

Comment:
I'd be inclined to split this into two tests: one to show the values that are recognized as true and another to show those that are not. It would save the reader from having to examine the data to see where the break comes - and let you use a simple loop for your test.